android/build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.0'
```

Comment:
oh, and I learned how to export the keymaps!

Go to `File -> Export Settings`, select only the keymaps, it'll create a .jar, then in Android Studio, `File -> Import Settings`.